server/sonar-server/src/main/java/org/sonar/server/settings/ws/ValuesAction.java

```diff
@@ -0,0 +1,291 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
```

Comment:
I think we're used to have boolean field without the is, so it should be `default` instead of `isDefault`